Twitter_API/code.py

```diff
@@ -14,11 +14,11 @@
 
 #pylint:disable=invalid-name
 import time
-import binascii
+import adafruit_binascii as binascii
```

Comment:
@dglaude Since we're using `adafruit_binascii`, could you please create a commit removing the old `binascii` file in this directory as to not confuse users: https://github.com/adafruit/Adafruit_Learning_System_Guides/blob/933c73cdefd097dbadc58ace9a865272d2a21298/Twitter_API/binascii.py